### PR TITLE
Preparing tests for LZ4-based references

### PR DESCRIPTION
**Pull request by oshadura**
Tests: tree-cloning was moved to CMake and was added LZ4 references; tree-cache, io-treeForeign,io-filemerger and io-compression was moved to LZ4 references

### CODE REVIEW

File: root/io/compression/MainEvent.cxx [hunk 1]
```diff
@@ -305,10 +305,14 @@ int main(int argc, char **argv)
       int expectedSize = -1;
       if (comp == 0) expectedSize = 5534559;
       else if (comp == 101) expectedSize = 1254808;
-      else if (comp == 201) expectedSize = 1111230;
+      else if (comp == 208) expectedSize = 1111230;
       else if (comp == 301) expectedSize = 1265018;
-      else if (comp == 405) expectedSize = 1284420;
-      else if (comp == 5) expectedSize = 1208369;
+      else if (comp == 404) expectedSize = 1284420;
+#ifdef R__HAS_LZ4
```
Comment (pcanal) [inline root/io/compression/MainEvent.cxx]: Does R__HAS_LZ4 means R__HAS_DEFAULT_LZ4?

File: root/io/compression/MainEvent.cxx [hunk 2]
```diff
@@ -305,10 +305,14 @@ int main(int argc, char **argv)
       int expectedSize = -1;
       if (comp == 0) expectedSize = 5534559;
       else if (comp == 101) expectedSize = 1254808;
-      else if (comp == 201) expectedSize = 1111230;
+      else if (comp == 208) expectedSize = 1111230;
```
Comment (pcanal) [inline root/io/compression/MainEvent.cxx]: Surprisingly the compression level change but not the final size, is that correct?

Comment (oshadura) [reply root/io/compression/MainEvent.cxx]: Thank you, I updated references while running them zero tolerances.

File: root/io/compression/MainEvent.cxx (hunk 1 above)
Comment (oshadura) [reply root/io/compression/MainEvent.cxx]: Updated in both root.git and roottest.git.

**Pull request merged**